natlab/natlabd/natlabd.go

```diff
@@ -46,6 +46,7 @@ import (
 
 var (
 	listen  = flag.String("listen", "/tmp/qemu.sock", "path to listen on")
+	write   = flag.String("write", "/tmp/qemu_client.sock", "path to respond to (for dgram mode)")
```

Comment:
This part's unnecessary and precludes multiple VMs attaching to the same natlab (which is what natlab's all about: testing interop between two peers).

The `recvfrom` system call (`ReadFromUnix` in Go) will return the address of each message. I just verified that works.

So the first time we get a message from a mac address, we can just remember its address. This is what my old code was trying to do, but wasn't working at the time because the address was the same as the server's address.

So then the vnetMacHost client needs to not hard-code a unix socket address and instead make up a temporary file name for its client side for each instance that's run.

But then there's more work to do on the vnetMacHost side to support multiple VMs at once probably. At least snapshotting and restoring from snapshots.